### DISCUSSION TITLE
v1.0.0文档

### DISCUSSION THREAD

**Discussion by ProgramRipper** [Mrs4s/go-cqhttp#1103] [Q&A] [answered]
v1.0.0的API发生了许多的改动，但是目前文档尚未更新，对于我的开发有一定阻碍。请问可以告知下文档的更新进度和发布时间，以及v1.0.0的发布计划吗？

**Comment by ProgramRipper** [comment]
如果说文档的发布还有一段时间，那么可以简单地告知下相对于v0.9的改动，或者相对于onebot协议的不同吗？（顺便一提，我在开发[avilla](https://github.com/GraiaProject/avilla-miraigo-onebot)对go-cqhttp的支持，基于其对onebot协议的支持）

**Comment by ProgramRipper** [comment, marked as answer]
emmm...一年了

**Reply by Mrs4s** [reply]
应该没啥变动的...

**Reply by ProgramRipper** [reply]
？
好吧，那现在release blocker是什么呢？onebot12吗？

**Reply by Mrs4s** [reply]
还在搓webui
